pages/common/calc.md

```diff
@@ -1,12 +1,36 @@
 # calc
 
-> An interactive arbitrary-precision calculator on the terminal.
+> An interactive arbitrary-precision calculator in the terminal.
 > More information: <https://github.com/lcn2/calc>.
 
-- Start calc in interactive mode:
+- Start `calc` in interactive mode:
 
 `calc`
 
-- Perform a calculation in non-interactive mode:
+- Perform a calculation in non-interactive mode and exit:
```

Comment:
```suggestion
- Perform a calculation in non-interactive mode:
```

If we assume that smth in non-interactive mode is exited by default then `and exit` is not necessary.